src/actions/registration/index.ts

```diff
@@ -36,9 +36,7 @@ export const createIdentity = (encodedEntropy: string): ThunkAction => async (
   dispatch(setLoadingMsg(loading.loadingStages[0]))
 
   const seed = await generateSecureRandomBytes(16)
-  const password = (await generateSecureRandomBytes(32)).toString('base64')
-  // const identity = await sdk.createNewIdentity(password)
-  const identity = await sdk.loadFromMnemonic(entropyToMnemonic(seed), password)
+  const identity = await agent.loadFromMnemonic(entropyToMnemonic(seed))
```

Comment:
This is more on the SDK side, but should there be a `loadFromSeed` method on the `Agent` instead?

Reply:
The JS code is never supposed to generate seeds or mess with them, but this code is here this way because we are missing support for hdkey derivation in the rust core....